applications/nrf_desktop/src/modules/dfu_mcumgr.c

```diff
@@ -4,8 +4,15 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#include <zephyr/kernel.h>
+
+#if defined CONFIG_DESKTOP_DFU_BACKEND_MCUBOOT
```

Comment:
I think we don't need this #if defined statement here as these MCUboot headers shouldn't have any impact on the SUIT backend

Reply:
Then I get error: fatal error: bootutil/image.h: No such file or directory
I should then add some more konfigs to include extra files

Reply:
I see it's an issue with the following header:

https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h

that includes the MCUboot repository headers when the relevant Kconfig is enabled

Reply:
To sum up just the `<zephyr/mgmt/mcumgr/grp/img_mgmt/img_mgmt.h>` requires the ifdef statement

Reply:
Ok, then it seems that we need to use ifdefs (at least for now)